test/testdata/golden/gapic/lang/csharp_gapic.yaml

```diff
@@ -0,0 +1,5 @@
+type: com.google.api.codegen.ConfigProto
```

Comment:
I tried to submodule the repo and realize that the output_dir and gapic output dir is not consistent. In this test, I am looking for a artman yaml config that is tailed for testing purpose.